modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -91,6 +92,276 @@ for alpha components and Preview as a space for beta components.
+| `h4` (16px)       | `levels.body.small` (1rem \ 16px)        |
+| `h5` (16px)       | `levels.body.small` (1rem \ 16px)        |
+| `body` (14px)     | `levels.subtext.large` (0.875rem \ 14px) |
+| `body2` (13px)    | `levels.subtext.small` (0.75rem \ 12px)  |
+| `small` (12px)    | `levels.subtext.small` (0.75rem \ 12px)  |
+
+- 🤖 Property Updates
+
+All `fontFamily`, `fontSize`, and `fontWeight` property updates are handled by the codemod.
+
+| CSS Property | Corresponding Token           | Notes                                                           |
+| ------------ | ----------------------------- | --------------------------------------------------------------- |
+| `fontFamily` | `type.properties.fonts`       | `normal` (Roboto) and `monospace` (Roboto Mono) are available   |
+| `fontSize`   | `type.properties.fontSizes`   | please consult the type hierarchies above to map values         |
+| `fontWeight` | `type.properties.fontWeights` | `regular` (400), `medium` (500), and `bold` (700) are available |
+
+- 🤖 Variant Updates
+
+All `variant` updates _except `link`_ are handled by the codemod. Please see the
+[variants](#variants) section below for more information.
+
+| Variant                | Transformation                                                                 | Notes                                   |
+| ---------------------- | ------------------------------------------------------------------------------ | --------------------------------------- |
+| `type.variant.error`   | `type.variants.error`                                                          | name change only                        |
+| `type.variant.hint`    | `type.variants.hint`                                                           | name change only                        |
+| `type.variant.inverse` | `type.variants.inverse`                                                        | name change only                        |
+| `type.variant.button`  | `{fontWieight: type.properties.fontWeights.bold}`                              | variant deprecated, use type properties |
+| `type.variant.caps`    | `{textTransform: 'uppercase', fontWeight: type.properties.fontWeights.medium}` | variant deprecated, use type properties |
+| `type.variant.label`   | `{fontWeight: type.properties.fontWeights.medium}`                             | variant deprecated, use type properties |
+| `type.variant.mono`    | `{fontWeight: type.properties.fonts.monospace}`                                | variant deprecated, use type properties |
+
+#### Manual Updates
+
+There are only two type deprecations not covered by the codemod:
+
+- Type Wrapper components (`H1`-`H5`) have been removed
+- `link` variant has been removed.
+
+##### Type Wrapper Components Migration
+
+To migrate, please refer to the hierachy tables about and use the type hierarchy tokens directly. Detailed usage information is available in the [levels section](#levels).
```

Comment:
Is "hierarchy tables" meant to refer to the tables in the [first list item under Automatic Updates](https://github.com/Workday/canvas-kit/blame/96fda328962cc43b4a697862fc62ba113cc41207/modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx#L106) above? If so you could make the change below to link directly to the tables:

```suggestion
##### Type Wrapper Components Migration

To migrate, please refer to the [old hierarchy tables](#old-hierarchy-tables) and use the type
hierarchy tokens directly. Detailed usage information is available in the [levels section](#levels).
```

You would also need to add an empty anchor tag to the destination line. Something like this:
```
- 🤖 Type Hierarchy Updates<a name="old-hierarchy-tables"></a>
```
It's a bit ugly, unfortunately. Alternatively, you could just link to the nearby `Automatic Updates` heading which will get you close enough (though this link may break if you move content around).